frontend/app/src/components/accounts/management/AccountForm.vue

```diff
@@ -31,6 +31,38 @@ const errors = defineModel<ValidationErrors>('errorMessages', { required: true }
 const chain = useRefPropVModel(modelValue, 'chain');
 
 const { isEvm } = useSupportedChains();
+const { t } = useI18n();
+const { apiKey, load: loadApiKeys } = useExternalApiKeys(t);
+const router = useRouter();
+
+const etherscanApiKeyAlert = computed(() => {
+  const selectedChain = get(chain);
+
+  if (selectedChain && selectedChain !== 'multievm' && get(isEvm(selectedChain))) {
```

Comment:
does the `multievm` option exist? it should be an `evm` property that exists and is true in the `AccountManageState`.